client/test/unit/specs/transactions.spec.js

```diff
@@ -312,11 +312,11 @@ describe('Testing Transactions.vue EXPIRED', () => {
     // set NR info
     vm.$store.commit('setNrInfo', {
       additionalInfo: 'The quick brown fox...',
-      consent_dt: '2021-08-01T07:01:00+00:00',
+      consent_dt: '2021-08-01T06:59:00+00:00',
       consentFlag: 'R',
       corpNum: 'BC1234567',
       entity_type_cd: null,
-      expirationDate: '2021-09-01T07:01:00+00:00',
+      expirationDate: '2021-09-01T06:59:00+00:00',
```

Comment:
In September we are in still in summer time, so UTC offset should be 8h. In other words, this expiration time should be 07:59.

Ref: https://www.timeanddate.com/time/zone/usa/los-angeles

Reply:
ok that makes sense...god i hate daylight savings time :(

Reply:
It's a rite of passage 😆

Reply:
haha done.  i also set consent time to 07:00 in line 315  which should be 0:00 in DB

Reply:
> i also set consent time to 07:00 in line 315 which should be 0:00 in DB

I think it's actually the other way around... should be:
- 00:00 Pacific time
- 08:00 UTC (note that this is summer time offset because the date is Aug 1)